chrome-extensions.md

```diff
@@ -0,0 +1,12 @@
+# Google Chrome Extensions 🛠
+
+Useful Chrome extensions extensions he community. Please, submit a Pull Request or Open an issue for a useful extension that you know.
```

Comment:
add these lines:

- [Tools](README.md)
- [VSCode Extensions](vscode-extensions.md)
- [Code Playgrounds](code-playgrounds.md)
- [Commands](commands.md)

and this line on the other files:

- [Chrome Extensions](chrome-extensions.md)

Reply:
proposed changes made